### PR TITLE
UI: Adjust input mapping view

### PR DESCRIPTION
**Pull request by nreinartz**
In the process of doing #4865 I took the liberty of adjusting the mapping view a little bit. I think most of the issues stem from the boxes not being aligned and the noise created by the amount of borders and small gaps between boxes.

Changes:
- Top most bar is now a 2x2 grid to create a better overview
- Most boxes are now properly aligned to their neighbours
- The "double borders" have been removed bei either joining borders or putting a margin between boxes
- Right controller buttons are now a list, just like DPad
- Separator between stick mappings and checkboxes
- Deadzone & Range headers are centered
- Move - and + to the bottom of their boxes

Fixes:
- Slider text blocks now have a fixed width to prevent the slider from changing size when value changes from something like 0.11 to 0.23 (different text widths causing the slider to get bigger/smaller).

Preview:

### CODE REVIEW

**Comment by nreinartz** [comment]
> Just FYI I have a pr in works for this already #4251

That's unfortunare, I must have overlooked that one. I'm fine either way, close it or use it as a middle way until yours is finished.

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 1]
```diff
@@ -21,35 +21,57 @@
...
     </UserControl.Resources>
+    <UserControl.Styles>
+        <Style Selector="ToggleButton">
+            <Setter Property="Width" Value="90" />
+            <Setter Property="Height" Value="27" />
+            <Setter Property="HorizontalAlignment" Value="Stretch" />
+        </Style>
+    </UserControl.Styles>
+    
+
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:32]:
```suggestion
```

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 2]
```diff
@@ -21,35 +21,57 @@
...
                     <TextBlock
-                        Margin="0,0,0,4"
-                        HorizontalAlignment="Center"
+                        Margin="5,0,10,0"
+                        Width="90"
+                        HorizontalAlignment="Left"
+                        VerticalAlignment="Center"
                         Text="{locale:Locale ControllerSettingsPlayer}" />
+
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:69]:
```suggestion
```

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 3]
```diff
@@ -21,35 +21,57 @@
...
-                    HorizontalAlignment="Center"
-                    VerticalAlignment="Center"
-                    Orientation="Vertical">
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Center">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:62]:
```suggestion
```

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 4]
```diff
@@ -59,69 +81,149 @@
...
-                    Orientation="Vertical">
+                    VerticalAlignment="Center">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="Auto"/>
+                    </Grid.ColumnDefinitions>
+
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:105]:
```suggestion
```

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 5]
```diff
@@ -59,69 +81,149 @@
...
+                            ToolTip.Tip="{locale:Locale ControllerSettingsRemoveProfileToolTip}"
+                            Command="{Binding RemoveProfile}">
+                        <ui:SymbolIcon
+                            Symbol="Delete"
+                            FontSize="15"
+                            Height="20" />
+                    </Button>
+                </Grid>
             </Border>
+            <!-- Input device -->
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:159]:
```suggestion

            <!-- Input device -->
```

Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:159]:
```suggestion

            <!-- Input device -->
```

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 6]
```diff
@@ -59,69 +81,149 @@
...
+                BorderBrush="{DynamicResource ThemeControlBorderColor}"
+                BorderThickness="1"
+                Padding="2">
+                <Grid Margin="2" HorizontalAlignment="Stretch">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="*"/>
+                        <ColumnDefinition Width="Auto" />
+                    </Grid.ColumnDefinitions>
+
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:173]:
```suggestion
```

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 7]
```diff
@@ -59,69 +81,149 @@
...
+                    <TextBlock
+                        Grid.Column="0"
+                        Margin="5,0,10,0"
+                        Width="90"
+                        HorizontalAlignment="Left"
+                        VerticalAlignment="Center"
+                        Text="{locale:Locale ControllerSettingsInputDevice}" />
+
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:181]:
```suggestion
```

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 8]
```diff
@@ -59,69 +81,149 @@
...
+                    <ComboBox
+                        Grid.Column="1"
+                        Name="DeviceBox"
+                        HorizontalAlignment="Stretch"
+                        VerticalAlignment="Center"
+                        Items="{Binding DeviceList}"
+                        SelectedIndex="{Binding Device}" />
+
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:189]:
```suggestion
```

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml [hunk 9]
```diff
@@ -59,69 +81,149 @@
...
+                        VerticalAlignment="Center"
+                        Command="{Binding LoadDevices}">
+                        <ui:SymbolIcon
+                            Symbol="Refresh"
+                            FontSize="15"
+                            Height="20"/>
+                    </Button>
+                </Grid>
+            </Border>            
+            <!-- Controler type -->
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:203]:
```suggestion

            <!-- Controler type -->
```

**Review by AcK77 — CHANGES_REQUESTED** [review]
Just some style comments. I really love it, it's looks way better than before. Thanks for working on it.

File: src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml (hunk 5 above)
Comment (nreinartz) [reply src/Ryujinx.Ava/UI/Windows/ControllerSettingsWindow.axaml:156]: I have the impression that github constantly duplicates these suggestions when I try to commit them. Weird.

**Review by AcK77 — APPROVED** [review]

**Review by marysaka — APPROVED** [review]

**Pull request merged**